utils/telldus-core/patches/980-auto-ptr.patch

```diff
@@ -0,0 +1,20 @@
+--- a/service/DeviceManager.cpp
```

Comment:
Why this patch is necessary? It is not mentioned in the commit message.

Reply:
minor fix for a compiler warning.

Reply:
Was this submitted to upstream? Why there is missing commit description in the patch itself? Why it is not mentioned in the commit description?

Reply:
Upstream is dead. Minor fixes are mentioned.

Reply:
In that case, we might want to remove this package. 

"Minor fixes" are too vague.

Reply:
But still it should be submitted to upstream first. Maybe it will be revived. Who knows. I don't like to carry patches forever.